src/SignalR/perf/benchmarkapps/Crankier/Crankier.csproj

```diff
@@ -4,6 +4,7 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <RootNamespace>Microsoft.AspNetCore.SignalR.CranksRevenge</RootNamespace>
+    <DisableTransitiveFrameworkReferences>true</DisableTransitiveFrameworkReferences>
```

Comment:
Why is this necessary? It doesn't look like it's something that's used often.

Reply:
[Version conflict](https://github.com/Azure/azure-signalr/issues/641) with Azure SignalR SDK

Reply:
Do you know if this is still necessary?

Reply:
Do you know if this is still necessary?

Reply:
Seems to still not build without it :/